test/e2e/autoscaling_utils.go

```diff
@@ -31,10 +31,11 @@ const (
 	consumptionTimeInSeconds = 30
 	sleepTime                = 30 * time.Second
 	requestSizeInMillicores  = 100
+	requestSizeInMegabytes   = 100
```

Comment:
My intuition here is that this value should be higher but let's leave it as a 'stabilization' work.